src/test/java/com/znsio/workflow/createUpdate.feature

```diff
@@ -0,0 +1,24 @@
+@prod @CreateUpdate
+  Feature: Create update and validate
+
+    @createPost
+    Scenario: Create and validate post
+      * def title = generateAlphaNumericRandomString(5)
+      * def body = generateAlphaNumericRandomString(10)
+      * def userId = generateRandomNumber(2)
+      Given def createPostResponse = karate.call('classpath:com/znsio/templates/jsonPlaceHolderTemplates.feature@t_createPost',{"title": title,"body": body,"userId": userId}).response
+      And print createPostResponse
+      * match createPostResponse.title == title
+      * match createPostResponse.body == body
+      * match createPostResponse.userId == userId
+      And assert createdPostResponse.id > 100
```

Comment:
Assert might not be the last validation. you may need to move it above or change it to match